wee_slack.py

```diff
@@ -1962,6 +1959,12 @@ def get_typing_list(self):
                 del self.typing[user]
         return typing
 
+    def mark_read(self, ts=None, update_remote=True, force=False, post_data={}):
+        if config.thread_messages_in_channel and not update_remote:
```

Comment:
The `not` shouldn't be here, it is when `update_remote` is true that we should mark thread channels (I used a bit awkward wording in the last comment, and said we should *not* do it if `update_remote` is false).